_pyinstaller_hooks_contrib/stdhooks/hook-niquests.py

```diff
@@ -0,0 +1,20 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2023 PyInstaller Development Team.
```

Comment:
I missed this earlier, but the year should be 2025.

Reply:
done